packages/dashboard/test/dashboard-widget-resizing.test.ts

```diff
@@ -323,6 +349,99 @@ describe('dashboard - widget resizing', () => {
       ]);
     });
 
+    (isSafari ? it.skip : it)('should resize only when dragged in the same direction (top -> bottom)', async () => {
```

Comment:
For some reason this test fails on Safari. When manually testing, it seems to work.